include/atomicassets.hpp

```diff
@@ -233,12 +233,6 @@ CONTRACT atomicassets : public contract {
         ATTRIBUTE_MAP new_data
     );
 
-    ACTION logbackasset(
```

Comment:
I think it probably makes sense to keep this in just so the new abi can still be used to deserialize old log actions